lib-rca/README.md

```diff
@@ -0,0 +1,2 @@
+# Root Cause Analysis (RCA) Java Library
+A Java library which analyzes statistics and provides root cause analysis (RCA) for clients.
```

Comment:
Can you add another line like "This is a work in progress." to be clear?

Reply:
Yes